app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -99,12 +99,29 @@ class RegisterController @Inject() (val helpToSaveService:     HelpToSaveService
           logger.warn(s"Error while trying to create account: ${submissionFailureToString(e)}", nino)
           SeeOther(routes.RegisterController.getCreateAccountErrorPage().url)
         },
-          _ ⇒ SeeOther(frontendAppConfig.nsiManageAccountUrl)
+          submissionSuccess ⇒ submissionSuccess.accountNumber.fold(
+            SeeOther(frontendAppConfig.nsiManageAccountUrl)
+          ) {
+              acNumber ⇒
+                SeeOther(routes.RegisterController.getAccountCreatedPage().url).withCookies(
+                  Cookie("accountNumber", acNumber.accountNumber)
```

Comment:
i think we dont have to deal with cookies , lets chat

Reply:
ok